lib/tools/android-manifest.js

```diff
@@ -74,4 +81,60 @@ manifestMethods.packageAndLaunchActivityFromManifest = async function (localApk)
   }
 };
 
+manifestMethods.compileManifest = async function (manifest, manifestPackage, targetPackage) {
+  log.debug(`Compiling manifest ${manifest}`);
+  let {platform, platformPath} = await getAndroidPlatformAndPath();
+  if (!platform) {
+    return new Error("Required platform doesn't exist (API level >= 17)");
+  }
+  log.debug('Compiling manifest.');
+  try {
+    let args = ['package', '-M', manifest, '--rename-manifest-package',
+                manifestPackage, '--rename-instrumentation-target-package',
+                targetPackage, '-I', path.resolve(platformPath, 'android.jar'),
+                '-F', manifest + '.apk', '-f'];
+    await exec(this.binaries.aapt, args);
+    log.debug("Compiled manifest");
+  } catch (err) {
+    throw new Error(`Error compiling manifest. Original error ${err.message}`);
```

Comment:
`Original error: ${err.message}`

so we don't get things like "Original error could not open file". what? the error couldn't open a file?